docs(typescript): update example https server (#5389)
fastify/fastify: docs/Reference/TypeScript.md

   import path from 'path'
   import fastify from 'fastify'
   ```

> 2. Follow the following steps before setting up the Fastify https server 
> for create the `key.pem` and `cert.pem` files, as follows:

Reviewer: ```suggestion
2. Perform the following steps before setting up a Fastify HTTPS server 
to create the `key.pem` and `cert.pem` files:
```

   import path from 'path'
   import fastify from 'fastify'
   ```

> 2. Follow the official documentation of [Node.js in the https section ](https://nodejs.org/api/https.html#httpscreateserveroptions-requestlistener)

Reviewer 2: It doesn't seems correct.

https://web.archive.org/web/20201217160232/https://nodejs.org/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server/

You should provide the `openssl` comment.

Author: Okay then I'll add this link: 
https://web.archive.org/web/20201217160232/https://nodejs.org/en/knowledge/HTTP/servers/how-to-create-a-HTTPS-server/

 Is i leave the text as it was before

Reviewer 2: Just copy the commend in the link and write inside the doc.

Author: Updated description see here: https://github.com/fastify/fastify/pull/5389/commits/4ec3b6c30637603b4fe2b5321170fe9f7070c21a

Author: Lint Docs / Lint Markdown fails

Author: now it is ok
